examples/advanced/variables.tf

```diff
@@ -58,6 +58,12 @@ variable "log_analysis_target_region" {
   default     = null
 }
 
+variable "cloud_logs_target_region" {
```

Comment:
why do we have a separate input in the example for `cloud_logs_target_region`? I notice we have the same thing for `eventstreams_target_region` and `cos_target_region`. But we don't even use them in the tests.
I think its over complicating the example for no reason. Can we just use the 1 `region` variable, and ensure that the tests will only use a list of regions that is supported by all the services here

Reply:
I have added a single region variable as `atracker_target_region` and updated the test to use the regions supported by all the services.